packages/dnb-eufemia/src/extensions/payment-card/icons/index.js

```diff
@@ -4,7 +4,7 @@ import BankAxept from './BankAxept'
 import DNB from './DNB'
 import Expired from './Clock'
 import Blocked from './Padlock'
```

Comment:
```suggestion
import { padlock_medium as Blocked } from '../../../icons'
```
We should delete the `Padlock.tsx` file.

Reply:
will do!